Mathlib/LinearAlgebra/PerfectPairing.lean

```diff
@@ -57,7 +59,61 @@ protected def flip : PerfectPairing R N M where
 
 @[simp] lemma flip_flip : p.flip.flip = p := rfl
 
--- TODO `M` and `N` are both reflexive
+/-- The linear equivalence from `M` to `Dual R N` induced by a perfect pairing. -/
+@[simps]
```

Comment:
You might want to double check what it generates using `simps!`. Chances are `toDualLeft_symm_apply` is not desired.